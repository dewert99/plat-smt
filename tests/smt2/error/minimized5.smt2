(declare-sort U)
(declare-const a U)
(declare-const b U)
(check-sat-assuming ((= a b)))
(get-unsat-core)
(pop)
