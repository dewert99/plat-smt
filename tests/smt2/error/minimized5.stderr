The last command was not `check-sat-assuming` that returned `unsat` at 5:2
(get-unsat-core)
 ^

